releasenotes/notes/improve-controlled-UnitaryGate-5f3aa3d44034cfa7.yaml

```
@@ -0,0 +1,9 @@
+---
+upgrade_circuits:
+  - |
+    Previously the `control` method of UnitaryGate was utilizing `Isometry` for decomposition.
+    This improvement switches the synthesis to `qs_decomposition` which approximately halves the gate count.
```

What about something like this:

```suggestion
    The `control` method of  :class:`.UnitaryGate` now uses :func:`.qs_decomposition` instead of :class:`.Isometry` for decomposition. 
    This change reduces the number of gates by approximately 2x.
```

---

done.